improver/field_texture.py

```diff
@@ -245,9 +245,6 @@ def process(self, input_cube):
 
         ratios = ratios.merge_cube()
         thresholded = BasicThreshold(self.diagnostic_threshold).process(ratios)
-        field_texture = iris.util.squeeze(
-            collapsed(thresholded, "realization", iris.analysis.MEAN)
-        )
-        field_texture.remove_coord("realization")
-
+        field_texture = iris.util.squeeze(collapse_realizations(thresholded))
+        field_texture.data = np.float32(field_texture.data)
```

Comment:
It would be good to modify one of the `process` unit tests to add an assertion on the dtype, so that this is checked at the plugin level.

Reply:
Done. Added to metadata checks.

Reply:
Good work.  I think there are a few technical things here that should be tidied up in a follow-on ticket, namely:
- The `BasicThreshold` plugin should squeeze the output cube so that a single threshold is represented as a scalar coordinate (check metadata standard for this with @bjwheltor).
- Similar edits should be made to `set_up_test_cubes` if possible, although this may be difficult in the short term if it affects many unit tests
- The `collapsed` function in `cube_manipulation` is the better place to check datatype preservation, rather than casting to float32 after each call

These aren't changes that need to be made in this PR, but a reminder to myself to open some follow-on technical work.